package.json

```diff
@@ -121,6 +121,7 @@
     "fs-extra": "^9.0.0",
     "husky": "^3.1.0",
     "jest": "^24.7.1",
+    "jest-environment-node": "26.6.2",
```

Comment:
What is this package for?

Reply:
It is for avoiding the error on `integration-test` like in [here](https://app.circleci.com/pipelines/github/sifnoc/zkopru/126/workflows/299a1d1e-00cc-4827-a21e-2796aab47879/jobs/1364)

```
@zkopru/integration-test:     [1639572103296] ERROR (11063 on default-7dc9cf27-c0ba-4cde-9f21-2c86c625b80a): contracts/tx-utils - getting signed transaction failed : Error: Expected message to be an Uint8Array
```

Cause it from jest bug [issue #7780](https://github.com/facebook/jest/issues/7780#issuecomment-669828353). ​

We had been no error using the old web3 package on signing transactions in  `integration-test`. but in this  PR, there is an error on signing cause type checking

Alternatively, we may use the latest jest version to avoid TypeErrors.
(I tried and no type errors)but it makes fixing points on all test codes.

Reply:
Gotcha, maybe we can upgrade jest in a different PR